site/content/docs/user/quick-start.md

```diff
@@ -377,6 +377,32 @@ networking:
   ipFamily: ipv6
 ```
 
+#### Dual Stack clusters
+You can run dual stack clusters using `kind`, on kubernetes versions +1.16
+but first you need to [enable ipv6 in your docker daemon][docker enable ipv6].
+
+```yaml
+# a dual-stack cluster
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+networking:
+  ipFamily: DualStack
+nodes:
+- role: control-plane
+- role: worker
+- role: worker
+featureGates:
+  IPv6DualStack: true
```

Comment:
we should flip this feature gate automatically on kubernetes versions where it's gated and kind's networking.ipFamily has been set to dualstack

Reply:
(i.e. the user should not have to specify this field)

Reply:
there's an example of this for 1.11 and provisioning inside the kubeadm config package

Reply:
we are passing a bool with IPv6 , not the ipFamily, but we can add it before that (I guess)